_docs/tasks/metrics-logs.md

```diff
@@ -9,210 +10,300 @@ type: markdown
+   ---
+   # Configuration for logentry instances
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: logentry
+   metadata:
+     name: newlog
+     namespace: istio-config-default
+   spec:
+     severity: '"warning"'
+     timestamp: request.time
+     variables:
+       source: source.labels["app"] | source.service | "unknown"
+       user: source.user | "unknown"
+       destination: destination.labels["app"] | destination.service | "unknown"
+       responseCode: response.code | 0
+       responseSize: response.size | 0
+       latency: response.duration | "0ms"
+     monitored_resource_type: '"UNSPECIFIED"'
+   ---
+   # Configuration for a stdio handler
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: stdio
+   metadata:
+     name: newhandler
+     namespace: istio-config-default
+   spec:
+    severity_levels:
+      warning: 1 # Params.Level.WARNING
+    outputAsJson: true
+   ---
+   # Rule to send logentry instances to a stdio handler
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: rule
+   metadata:
+     name: newlogstdio
+     namespace: istio-config-default
+   spec:
+     match: "true" # match for all requests
+     actions:
+      - handler: newhandler.stdio
+        instances:
+        - newlog.logentry
+   ---
    ```
 
-   The expected output is:
+1. Push the new configuration.
 
    ```bash
-   Error: the server could not find the requested resource
+   kubectl -n istio-config-default apply -f new_telemetry.yaml
    ```
 
-   If your selected service has service-specific rules, update `new_rule.yaml`
-   to include the existing rules appropriately. Append the rule from `new_rule.yaml`
-   to the existing `rules` block and save the updated content back over `new_rule.yaml`.
-
-1. Push the new configuration to Mixer for a specific service.
-
-   ```bash
-   istioctl mixer rule create reviews.default.svc.cluster.local reviews.default.svc.cluster.local -f new_rule.yaml
+   Expected output:
+   ```
+   metric "doublerequestcount" configured
+   prometheus "doublehandler" configured
+   rule "doubleprom" configured
+   logentry "newlog" configured
+   stdio "newhandler" configured
+   rule "newlogstdio" configured
    ```
 
-1. Send traffic to that service.
+1. Send traffic to the sample application.
 
-   For the BookInfo sample, visit `http://$GATEWAY_URL/productpage` in your web browser or
-   issue the following command:
+   For the BookInfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+   browser or issue the following command:
 
    ```bash
    curl http://$GATEWAY_URL/productpage
    ```
 
-   For purposes of this task, please refresh the page several times or issue the curl
-   command a few times to generate traffic.
+   For purposes of this task, please refresh the page several times or issue the
+   curl command a few times to generate traffic.
+
+1. Verify that the new metric values are being generated and collected.
+
+   Setup port-forwarding for Prometheus:
```

Comment:
In a Kubernetes environment, setup ... using the following command:

Reply:
done.